src/verify/VerificationStartScreen.tsx

```diff
@@ -105,35 +111,33 @@ function VerificationStartScreen({
   }
 
   useLayoutEffect(() => {
-    const title = route.params?.hideOnboardingStep
-      ? t('phoneVerificationScreen.screenTitle')
-      : () => (
-          <HeaderTitleWithSubtitle
-            title={t('phoneVerificationScreen.screenTitle')}
-            subTitle={t('registrationSteps', { step, totalSteps })}
-          />
-        )
+    const title = () => (
+      <HeaderTitleWithSubtitle
+        title={t('phoneVerificationScreen.screenTitle')}
+        subTitle={showSteps && t('registrationSteps', { step, totalSteps })}
+      />
+    )
 
     navigation.setOptions({
       headerTitle: title,
       headerRight: () =>
-        !route.params?.hideOnboardingStep && (
+        route.params?.isOnboarding && (
           <TopBarTextButton
             title={t('skip')}
             testID="PhoneVerificationSkipHeader"
             onPress={onPressSkip}
             titleStyle={{ color: colors.onboardingBrownLight }}
           />
         ),
-      headerLeft: () => route.params?.hideOnboardingStep && <BackButton />,
+      headerLeft: () => !route.params?.isOnboarding && <BackButton />,
       // Disable iOS back during onboarding
-      gestureEnabled: route.params?.hideOnboardingStep ? false : true,
+      gestureEnabled: !route.params?.isOnboarding,
```

Comment:
This condition is flipped now. I believe this was buggy originally as we wanted to disable back gesture during onboarding but this was actually disabling them when not in onboarding